Maestro/subscriptions.json

```diff
@@ -885,6 +885,40 @@
         }
       }
     },
+    // Automate merging runtime release/8.0-rc branches back to release/8.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/runtime/blob/release/8.0-rc/**/*"
+      ],
+      "action": "github-dnceng-branch-merge-pr-generator",
+      "actionArguments": {
+        "vsoSourceBranch": "main",
+        "vsoBuildParameters": {
+          "GithubRepoOwner": "dotnet",
+          "GithubRepoName": "<trigger-repo>",
+          "HeadBranch": "<trigger-branch>",
+          "BaseBranch": "release/8.0",
+          "ExtraSwitches": "-QuietComments"
+        }
+      }
+    },
+    // Automate opening PRs to merge runtime release/8.0 branch back to main
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/runtime/blob/release/8.0//**/*"
+      ],
+      "action": "github-dnceng-branch-merge-pr-generator",
```

Comment:
Did we do 7 to main last time? Usually I see main and back port.

Reply:
aspnetcore does 8 -> main because we don't do backports during RCs, we just have people open 1 PR. Since runtime is doing main + backports, you probably don't need this one

Reply:
OK I'll remove it.